sci-biology/squizz/squizz-0.99b.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+DESCRIPTION="squizz is a sequence format file checker, but it has some
+conversion capabilities too."
```

Comment:
`DESCRIPTION` too long.

https://devmanual.gentoo.org/ebuild-writing/variables/
`DESCRIPTION` 	A short (less than 80 characters) description of the package's purpose. Mandatory.